ptp-tools/Makefile

```
@@ -23,37 +23,97 @@ endif
 
 podman-clean-%:
 	podman manifest rm ${IMG_PREFIX}:$* || true
+endif
 
 podman-cleanall: $(foreach v, $(VALUES), podman-clean-$(v))
 
+.PHONY: podman-buildall podman-pushall podman-saveall podman-cleanall
+.PHONY: podman-build-and-pushall podman-build-and-saveall
+.PHONY: build-image clean-image deploy-all
+
 deploy-all: 
```

checkmake (0.3.2)

[warning] Target body for "deploy-all" exceeds allowed length of 5 lines (7).

(maxbodylength)